demo/admin/src/common/MasterMenu.tsx

```diff
@@ -87,6 +87,7 @@ export const masterMenuData: MasterMenuData = [
                 );
             },
         },
+        requiredPermission: "pageTree",
```

Comment:
Should we add an enum in the library for this?

Reply:
77d66758ec67b49e0fa3843eb1ee27074aa4179e